phpcs.xml.dist

```diff
@@ -342,5 +342,15 @@
 		<exclude-pattern>/src/wp-content/themes/twentyeleven/inc/widgets\.php</exclude-pattern>
 		<exclude-pattern>/src/wp-content/themes/twentyfourteen/inc/widgets\.php</exclude-pattern>
 		<exclude-pattern>/src/wp-content/themes/twentyfourteen/inc/featured-content\.php</exclude-pattern>
+
+	</rule>
+
+	<!-- Add exception for test class not already whitelisted by WPCS. -->
+	<rule ref="WordPress.Files.FileName">
+		<properties>
+			<property name="custom_test_class_whitelist" type="array">
```

Comment:
Requires upstream change to extend this list:

https://github.com/WordPress/WordPress-Coding-Standards/blob/02af9f6dab171b5782d0945d994fce731915406d/WordPress/Sniff.php#L856-L876